src/main/test/com/epam/preprod/sirenko/dao/ProductDAOTest.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.preprod.sirenko.dao;
+
+import com.epam.preprod.sirenko.entity.Clothing;
+import com.epam.preprod.sirenko.entity.DryFood;
+import com.epam.preprod.sirenko.entity.Food;
+import com.epam.preprod.sirenko.entity.Product;
+import com.epam.preprod.sirenko.enums.PetGroup;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ProductDAOTest {
+	private static ArrayList<Product> allProducts;
```

Comment:
We dont need static here.Also we can use @BeforeEach instead of @beforeAll and remove  redundant variables

Reply:
changed